gflib/bg.c

```diff
@@ -695,7 +696,7 @@ s32 ChangeBgY(u8 bg, s32 value, u8 op)
     return sGpuBgConfigs2[bg].bg_y;
 }
 
-s32 ChangeBgY_ScreenOff(u8 bg, u32 value, u8 op)
+s32 ChangeBgY_ScreenOff(u8 bg, s32 value, u8 op)
```

Comment:
Maybe they should all be `u32`.  How do you know which is right?

Reply:
Others are s32 and the values being operated on are signed. Plus it makes no sense why a delta parameter should be unsigned